extensions/wikia/Email/EmailController.class.php

```diff
@@ -481,17 +481,34 @@ protected function getAvatarURL( \User $user ) {
 	}
 
 	protected function findUserFromRequest( $paramName, \User $default = null ) {
-		$userName = $this->getRequest()->getVal( $paramName );
-		if ( empty( $userName ) ) {
+		$user = $this->getRequest()->getVal( $paramName );
+		if ( empty( $user ) ) {
 			return $default;
 		}
 
 		// Allow an anonymous user to be specified
-		if ( $userName == self::ANONYMOUS_USER_ID ) {
+		if ( $user == self::ANONYMOUS_USER_ID ) {
 			return \User::newFromId( 0 );
 		}
 
-		return $this->getUserFromName( $userName );
+		if ( is_numeric( $user ) ) {
```

Comment:
Ah, good catch.

The reason I'm extending it to accept userId is because the ticket called for it. I'm going to update so that can be passed in as a separate parameter to avoid the ambiguity.